.github/workflows/update-package-lock.yaml

```diff
@@ -46,7 +43,7 @@ jobs:
             echo "No change."
           else
             npm test
-            npx hereby lkg
+            npx hereby LKG

```

Comment:
What's the point of the LKG? Just as a test to see that it works?

Reply:
E.g. the LKG script silently broke because a dep updated, or something. Otherwise we wouldn't know until we attempted to do a release or nightly or something.

Reply:
(this particular change is just to make all of the workflows have the same casing; silly, I know)